src/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+      <title>Hello</title>
+    </head>
+    <body>
```

Comment:
use 2 whitespaces instead of 4
```suggestion
  <body>
```